codewof/programming/content/en/duck-goose/question.md

```diff
@@ -0,0 +1,9 @@
+# Duck goose
+
+Write a function `duck_goose(string)` that returns `True` if the substrings `duck` and `goose` appear the same number of times in the given string, otherwise returns `False`.
+
+*Note: Python's built in function count() may be useful.*
+
+*Hint: You should convert the string to lowercase to handle capital letters.*
```

Comment:
> the substrings `duck` and `goose`

Are exactly that.
`DUCK` etc is not one of those substrings so I'd argue capital letters should not be handled. If you want them to be, then consider rewording the first sentence and removing this hint